base/shell/explorer/trayntfy.cpp

```diff
@@ -19,49 +19,24 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
-#include "precomp.h"
+#include "trayntfy.h"
 
-/*
- * TrayNotifyWnd
- */
-
-static const WCHAR szTrayNotifyWndClass[] = L"TrayNotifyWnd";
-
-#define TRAY_NOTIFY_WND_SPACING_X   1
-#define TRAY_NOTIFY_WND_SPACING_Y   1
-
-class CTrayNotifyWnd :
-    public CComCoClass<CTrayNotifyWnd>,
-    public CComObjectRootEx<CComMultiThreadModelNoCS>,
-    public CWindowImpl < CTrayNotifyWnd, CWindow, CControlWinTraits >,
-    public IOleWindow
-{
-    CComPtr<IUnknown> m_clock;
-    CComPtr<IUnknown> m_pager;
 
-    HWND m_hwndClock;
-    HWND m_hwndPager;
-
-    HTHEME TrayTheme;
-    SIZE szTrayClockMin;
-    SIZE szTrayNotify;
-    MARGINS ContentMargin;
-    BOOL IsHorizontal;
-
-public:
-    CTrayNotifyWnd() :
+    CTrayNotifyWnd::CTrayNotifyWnd() :
```

Comment:
I'm not a fan of these changes. Perhaps we may accept them in a different pull request but for now here it just clutters the real code changes.